Sources/TuistSigning/SigningInstaller.swift

```diff
@@ -1,37 +1,29 @@
 import Foundation
 import TSCBasic
+import TuistCore
 import TuistSupport
 
-enum SigningInstallerError: FatalError, Equatable {
-    case noFileExtension(AbsolutePath)
-    case provisioningProfilePathNotFound(ProvisioningProfile)
-    case revokedCertificate(Certificate)
-    case expiredProvisioningProfile(ProvisioningProfile)
-
-    var type: ErrorType {
-        switch self {
-        case .noFileExtension, .provisioningProfilePathNotFound, .revokedCertificate, .expiredProvisioningProfile:
-            return .abort
-        }
+extension LintingIssue {
+    static func noFileExtension(_ path: AbsolutePath) -> Self {
+        Self(reason: "Unable to parse extension from file at \(path.pathString)", severity: .error)
```

Comment:
this is due to a misconfiguration, so I think it can be treated as an error

Reply:
agree 👍